Assets/MixedRealityToolkit.Providers/WindowsMixedReality/WindowsMixedRealityDeviceManager.cs

```diff
@@ -373,7 +373,7 @@ public override void Disable()
             InteractionSourceState[] states = InteractionManager.GetCurrentReading();
             for (var i = 0; i < states.Length; i++)
             {
-                RemoveController(states[i]);
+                RemoveController(states[i].source);
```

Comment:
nit: may be worth adding null check

Reply:
Since this is coming from the platform, I think it's guaranteed to not be null.
@davidkline-ms, do you happen to know if the platform might ever provide null here?